project/dependencies.scala

```diff
@@ -78,16 +80,25 @@ object dependencies extends AutoPlugin {
     )
   }
 
+  private val tapir = Def.setting {
+    Seq(
+      "com.softwaremill.sttp.tapir" %% "tapir-core"               % V.tapir   % Provided,
+      "com.softwaremill.sttp.tapir" %% "tapir-openapi-docs"       % "0.16.16" % Test,
+      "com.softwaremill.sttp.tapir" %% "tapir-openapi-circe-yaml" % "0.16.16" % Test
+    )
+  }
+
   private val scalacheck = Seq(
     "org.scalacheck" %% "scalacheck" % V.scalacheck % Provided
   )
 
   private val documentation = Seq(
-    "org.typelevel"  %% "cats-effect" % "2.2.0",
-    "io.circe"       %% "circe-core"  % "0.13.0",
-    "org.tpolecat"   %% "doobie-h2"   % "0.9.2",
-    "org.http4s"     %% "http4s-dsl"  % "0.21.7",
-    "org.scalacheck" %% "scalacheck"  % "1.14.3"
+    "org.typelevel"               %% "cats-effect" % "2.2.0",
+    "io.circe"                    %% "circe-core"  % "0.13.0",
+    "org.tpolecat"                %% "doobie-h2"   % "0.9.2",
+    "org.http4s"                  %% "http4s-dsl"  % "0.21.7",
+    "org.scalacheck"              %% "scalacheck"  % "1.14.3",
+    "com.softwaremill.sttp.tapir" %% "tapir-core"  % "0.16.16"
```

Comment:
```suggestion
    "com.softwaremill.sttp.tapir" %% "tapir-core"  % V.tapir
```

Reply:
For tests we use the latest version, so Scala Steward updates it and we can ensure that the integration works with the latest version (if something breaks we get a failed PR)